codalab/server/rest_server.py

```diff
@@ -135,6 +136,9 @@ def wrapper(*args, **kwargs):
                 code, message = exception_to_http_error(e)
                 if code == INTERNAL_SERVER_ERROR:
                     traceback.print_exc()
+                    # Internal error details should not be echoed to client
```

Comment:
That's a good point-- I was worried about creating security issues by exposing internal error details, but is that perhaps not a concern since everything is open-source anyways?